### DISCUSSION TITLE
[Question] Missing macOS partiotions on Surface Book 3

### DISCUSSION THREAD

**Discussion by thaiiy** [badstorm/surface-pro-7-opencore#42] [Q&A]
Hi there,
I have a Surface Book 3 with i5 10th Gen CPU too.
After copying your EFI over, my USB drive containing my MacOS boots to the Opencore Boot Menu with only MS Windows as the partition available.

Did you have to change your BIOS settings to anything? I can't see many settings within the Surface BIOS, no AHCI settings to change.
Thanks

**Comment by badstorm** [comment]
This repo is specific only for Surface Pro. I don't have your device to make tests. I move this issue to discussions. Maybe some other users can help you.

**Comment by thaiiy** [comment]
I got it working thanks. I can't quite remember how, maybe MinVer -1 or something.
It was a random chance I got the booter quirks to get it through, now the Surface Book 3 is partly working. Keyboard and trackpad doesn't work, VoodooInput and VoodooI2CHID is not loading for some reason, even though the verbose boot says success. After boot, kextstat shows that they're not loaded. Any ideas?
